### ISSUE TITLE
False Positive: definition-list when role overriden

### ISSUE THREAD

**Issue by dsturley** [dequelabs/axe-core#129]
```html
<div role="list" id="target">
	<dl role="listitem">
	  <dd role="presentation">
			<a href="#somethin">Stuff</a>
		</dd>
	</dl>
</div>
```

Lists also have this problem:
```html
<ol role="presentation">
<span>hi</span>
</ol>
```

```html
<li role="presentation">
<span>hi</span>
</li>
```
